core/app/models/spree/payment.rb

```diff
@@ -13,7 +13,7 @@ class Payment < Spree::Base
     has_many :offsets, -> { offset_payment }, class_name: "Spree::Payment", foreign_key: :source_id
     has_many :log_entries, as: :source
     has_many :state_changes, as: :stateful
-    has_many :capture_events, :class_name => 'Spree::PaymentCaptureEvent'
+    has_many :capture_events, class_name: 'Spree::PaymentCaptureEvent'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.